docusaurus/docs/introduction/data-frames.md

```diff
@@ -87,6 +87,8 @@ Let's look at an example. The following table demonstrates a data frame with two
 
 Each field has three values, and each value in a field must share the same type. In this case, all values in the `time` field are timestamps, and all values in the `temperature` field are numbers.
 
+Time fields in the frontend and [backend](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/transformations/transformers/convertFieldType.ts#L231-L243) should be given in numeric values.
```

Comment:
Some comments here:

- The time type in the backend is `time.Time`, not a number
- The link you added is pointing to a function called `ensureTimeField`. This function can be used when the time (in the frontend) is not a number. This function attempts to convert the value to a number and it accepts:
  - Strings following the format ISO 8601
  - `DateTime` objects
  - Strings following the format `now` and a relative time (e.g. `now-10s`)
 
Also note that the link is not properly pointing to the function. I don't know if there is a policy about this but links in code should not point to `blob/main` since that link will point to different code as the codebase evolves. Better to use a specific commit or branch (like https://github.com/grafana/grafana/blob/v10.1.5/packages/grafana-data/src/transformations/transformers/convertFieldType.ts#L231-L243).

So because the above, I suggest to change the new paragraph a bit:

> While time fields represent timestamps, the type of the values should be `Number` (TypeScript) or `time.Time` (Golang).
>
> In the plugin frontend code, it's possible to convert other formats to `Number` using the function [`ensureTimeField`](https://github.com/grafana/grafana/blob/v10.1.5/packages/grafana-data/src/transformations/transformers/convertFieldType.ts#L231-L243) from the `@grafana/data` package. This function converts strings following the ISO 8601 format (e.g. `2017-07-19 00:00:00.000`), Javascript `DateTime`s and strings with relative times (e.g. `now-10s`) to `Number`s.

Reply:
my latest commit makes the chnage you suggest. thanks for the corrections!